std/typecons.d

```diff
@@ -3894,7 +3998,7 @@ if (Targets.length >= 1 && allSatisfy!(isMutable, Targets))
     {
         auto wrap(inout Source src)
         {
-            static assert(hasRequireMethods!(),
+            static assert(implementsInterface!(Source, Targets),
                           "Source "~Source.stringof~
                           " does not have structural conformance to "~
                           Targets.stringof);
```

Comment:
I'll look into something in the future. I'm expecting that using only one or two targets is most common, and I'm not sure what it will take to specify what method wasn't conforming.